docs/Documentation/Drivers/Classic/OutputFormatting.md

```diff
@@ -59,7 +53,7 @@ _type_ = (for binary output files) data type code. Must be one of:
   - `OUT_TYPE_CHAR` = char
   - `*` = use the default type
 
-_multiplier_ = (for binary output files) factor to multiply the data by before writing, to increase precision.
+_multiplier_ = (for `BINARY` output files) factor to multiply the data by before writing, to increase precision.
```

Comment:
This could be a bit misleading because we don't "increase precision" compared to what's done internally in the code. I suggest changing to (we increase it compared to not using the multiplier):
```
+_multiplier_ = (for `BINARY` output files) factor to scale the data by before writing to minimize loss of precision.
```